src/app/tests/TestICDManager.cpp

```diff
@@ -319,28 +317,21 @@ class TestICDManager
 } // namespace chip
 
 namespace {
-/**
- *   Test Suite. It lists all the test functions.
- */
-// clang-format off
-static const nlTest sTests[] =
-{
-    NL_TEST_DEF("TestICDModeDurations",         TestICDManager::TestICDModeDurations),
-    NL_TEST_DEF("TestKeepActivemodeRequests",   TestICDManager::TestKeepActivemodeRequests),
-    NL_TEST_DEF("TestICDMRegisterUnregisterEvents", TestICDManager::TestICDMRegisterUnregisterEvents),
-    NL_TEST_SENTINEL()
+static const nlTest sTests[] = {
+    NL_TEST_DEF("TestICDModeDurations", TestICDManager::TestICDModeDurations),
+    // NL_TEST_DEF("TestKeepActivemodeRequests",   TestICDManager::TestKeepActivemodeRequests),
+    // NL_TEST_DEF("TestICDMRegisterUnregisterEvents", TestICDManager::TestICDMRegisterUnregisterEvents),
+    NL_TEST_SENTINEL(),
```

Comment:
Why are these commented out? Could it be a mistake?

Reply:
Thanks for spotting that! Fixed